crates/rattler/src/cli/auth.rs

```diff
@@ -24,6 +24,18 @@ pub struct LoginArgs {
     /// The token to use on anaconda.org / quetz authentication
     #[clap(long)]
     conda_token: Option<String>,
+
+    /// The S3 access key ID
+    #[clap(long)]
+    s3_access_key_id: Option<String>,
```

Comment:
We can mark these as mutually exclusive with `conda_token` etc. See https://docs.rs/clap/latest/clap/struct.Arg.html#method.conflicts_with. Similarly we can also use [`required_if`](https://docs.rs/clap/latest/clap/struct.Arg.html#method.requires_if) to link the key id and access together.